python/ray/util/client/worker.py

```diff
@@ -462,7 +468,7 @@ def _server_init(self, job_config: JobConfig):
             import ray._private.runtime_env as runtime_env
             import tempfile
             with tempfile.TemporaryDirectory() as tmp_dir:
-                (old_dir, runtime_env.PKG_DIR) = (runtime_env.PKG_DIR, tmp_dir)
+                runtime_env.PKG_DIR = tmp_dir
```

Comment:
I'm a bit confused as to why we don't want to restore `runtime_env.PKG_DIR` anymore.

Reply:
Although this will work and when running to this line, we usually will have `PKG_DIR == None`, I'd go for a restore since it makes things look hard to understand here. 
Besides, after later users might be able to upload the runtime env by themselves, and they'll need to set up the PKG_DIR. So to make things safe, let's restore it.